src/content/ui/adaptive-responsive/large-screens.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Large screen devices
+description: >
+  xxx
+short-title: Large screens
+---
+
+<b>TODO(sfshaza): PENDING: Should we axe this page?</b>
```

Comment:
+1 to axing it, I don't think we have any specific large screen guidance other than to follow what we've covered in the other sections already.

Reply:
shams and I discussed this offline and she plans to remove it, good to know you agree!

Reply:
If we kept this page I think it should have a brief blurb about how large screen should be supported with the adaptive content then have a section about how to avoid letterboxing on android.

Reply:
I have completely rewritten EVERYTHING, based on your talk!